model-optimizer/automation/package_BOM.txt

```diff
@@ -538,6 +538,8 @@ extensions/middle/MulFakeQuantizeFuse.py
 extensions/middle/MXNetRNNSequenceNormalize.py
 extensions/middle/MXNetSplitMultiLayers.py
 extensions/middle/MXTileReplacer.py
+extensions/middle/ONNXResize11ToInterpolateV4.py
+extensions/middle/ONNXResize11WithTFCropAndResizeConversion.py
```

Comment:
Please, rename the file and the class name to `ONNXResize11ToTFCropAndResize`

Reply:
Done.